topics/statistics/tutorials/classification_regression/tutorial.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: tutorial_hands_on
+
+title: "Machine learning: classification and regression"
+questions:
+  - "what are classification and regression techniques?"
+  - "How they can be used for prediction?"
+  - "How visualizations can be used to analyze predictions?"
+objectives:
+  - "Explain the types of supervised machine learning - classification and regression."
+  - "Learn how to make predictions using the training and test data."
+  - "Visualize the predictions."
+requirements:
```

Comment:
You probably need Galaxy introduction as requirements

Reply:
It is there, the galaxy introduction because it is under "statistics" tool category and this category already has it